code/WorkInProgress/Cael_Aislinn/sculpture.dm

```diff
@@ -144,7 +144,7 @@
 		spawn()
 			while(get_turf(src) != target_turf && num_turfs > 0)
 				for(var/obj/structure/window/W in next_turf)
-					W.destroy()
+					W.Destroy()

```

Comment:
And I don't think Destroy calls del, since del calls Destroy. Have you tested this?